msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -864,129 +864,129 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
+|`tabs.groups.controls.items.supertip.title`| String | 64 characters | ✔️ | Specifies the title text of the supertip.|
+|`tabs.groups.controls.items.supertip.description`| String | 128 characters | ✔️ | Specifies the description of the supertip.|
+|`tabs.groups.controls.items.actionId`| String | 64 characters | ✔️ | Specifies the ID of the action that is taken when a user selects the control or menu item. The `actionId` must match with `runtime.actions.id`. |
+|`tabs.groups.controls.items.enabled`| Boolean | | | Indicates whether the control is initially enabled. Default value: `true`.|
 |`tabs.groups.controls.items.overriddenByRibbonApi`| Boolean | | | Specifies whether a group, button, menu, or menu item hidden on application and platform combinations that support the API ([Office.ribbon.requestCreateControls](/javascript/api/office/office.ribbon#office-office-ribbon-requestcreatecontrols-member(1))) that installs custom contextual tabs on the ribbon. Default value: `false`.|
 |`tabs.groups.controls.type`| String | | ✔️ | Defines the control type. Supported values: `button`, `menu`.|
-|`tabs.groups.controls.builtinControlId`| String | 64 characters | ✔️ | ID of the existing Microsoft 365 control. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
-|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Text displayed for the control.|
+|`tabs.groups.controls.builtinControlId`| String | 64 characters | ✔️ | Specifies the ID of the existing Microsoft 365 control. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
+|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control.|
 |`tabs.groups.controls.icons`| Array | | | Defines the icon(s) for the control. |
-|`tabs.groups.controls.icons.size`| Number | | ✔️ | Size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Required image size: `16`, `32`, `80`. |
-|`tabs.groups.controls.icons.url`| URL| | | URL to the icon.|
-|`tabs.groups.controls.icons.file`| String | 2048 characters| | Relative path to the file that contains the icon. This property isn’t supported as a relative path. |
+|`tabs.groups.controls.icons.size`| Number | | ✔️ | Specifies the size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Required image size: `16`, `32`, `80`. |
+|`tabs.groups.controls.icons.url`| URL| | | Specifies the URL to the icon.|
+|`tabs.groups.controls.icons.file`| String | 2048 characters| | Specifies the relative path to the file that contains the icon. This property isn’t supported as a relative path. |
```

Comment:
Update the `url` property to be "Specifies the absolute URL of the icon." 
Remove the `file` property entirely.